api/v1/openapi.yaml

```diff
@@ -2875,6 +2878,24 @@ definitions:
       users:
         description: number of users of this selector in the cache
         type: integer
+  SRv6:
+    description: |-
+      Status of the SRv6
+
+      +k8s:deepcopy-gen=true
+    type: object
+    properties:
+      enabled:
+        type: boolean
+      devices:
+        type: array
+        items:
+          type: string
```

Comment:
Why do we need to have devices here?

Reply:
I just followed what's in the issue description, could you please give an advise on this?
Shall I remove the devices?

Reply:
Removed the devices

Reply:
We're getting rid of the device dependency and will soon deprecate it. So, it's no longer required. Sorry, I should explain this background first. Anyways, thanks!